src/odemis/driver/smaract.py

```diff
@@ -2747,6 +2794,7 @@ def _doMoveRel(self, future, pos):
             CancelledError: if cancelled before the end of the move
         """
         with future._moving_lock:
+<<<<<<< HEAD
```

Comment:
I think this is a result of the same rebase/merge/pull which is not yet completed